### DISCUSSION TITLE
What is the best method to sort boolean columns

### DISCUSSION THREAD

**Discussion by carbtest** [rappasoft/laravel-livewire-tables#1746] [Q&A] [answered]
with the following code 
```
...,
BooleanColumn::make("Allow Multiple", 'allow_multiple')
    ->sortable(),
...

 public function builder(): Builder
    {
        return Field::query()->orderBy('name', 'asc');
    }
```
what is the best way to make the column sortable?

**Comment by lrljoe** [comment]
Don't set an orderBy in builder, as that will override any other sorts you're doing.

With that removed, your sortable() will work exactly as is

**Comment by carbtest** [comment]
What if I want an initial sort?

Joe Bradfield
***@***.***
________________________________

**Reply by lrljoe** [reply, marked as answer]
Perfectly possible 

```
public function configure(): void
{
    $this->setDefaultSort('name', 'desc');
}
```

There's a ton of customisations etc, and they're fairly well documented, for example, the doc for setting a default sort is:
[See Docs](https://rappasoft.com/docs/laravel-livewire-tables/v3/sorting/available-methods#content-setdefaultsort)
